src/SFA.DAS.EmployerCommitmentsV2.Web/Views/Shared/_ZenDeskWidget.cshtml

```diff
@@ -11,10 +19,11 @@
             },
             helpCenter: {
                 filter: {
-                    section: "360001841359"
+                    section: @ZenDeskConfiguration.SectionId
                 }
             }
         }
     };
 </script>
-<script id="ze-snippet" src="https://static.zdassets.com/ekr/snippet.js?key=1b1ce288-39ea-4d40-a5e1-3a5e08aadd89"> </script>
\ No newline at end of file
+<script id="ze-snippet" src="@snippetUrl"></script>
+@Html.SetZendeskLabels($"App-{ViewBag.Title}")
```

Comment:
As in previous PR is Title appropriate here what if the title is dynamic ie "John Smith Details...."